governance/roles.md

```diff
@@ -159,9 +172,12 @@ goals, the meeting facilitator has the following responsibilities:
 Prerequisites:
 * active member
 * history of regular attendance
-* prior participation as scribe
+
+Preferred prerequisites:
+* prior experience scribing for a meeting
 
 One or more of the following:
+* active member of the SIG
```

Comment:
"active member" listed above -- I believe it should be a requirement, not a preferred prereq